vaadin-testbench-shared/src/main/java/com/vaadin/testbench/Parameters.java

```diff
@@ -398,7 +398,7 @@ public static boolean isLocalWebDriverUsed() {
     /**
      *
      * Gets the maximum number of times to run the test in case of a random
-     * failure. See {@link RetryRule} for details.
```

Comment:
Would it be helpful to reference`getMaxAttempts` from `RetryRule`?

Reply:
It can't be referred as `RetryRule` is in `vaadin-testbench-core` module and there is no such feature in `vaadin-testbench-core-junit5`.